src/components/MavenDependency.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function MavenDependency({gav, hasBomEntry} ) {
+    const [group, artifactId, version] = gav.split(':');
+    return (
+        <>
+            <h3>Declaring a dependency</h3>
+            <p>As a plugin developer you can use this plugin as dependency of your plugin by adding a dependency tag to your POM.</p>
+            {!hasBomEntry && <p>To add the latest version of this plugin as a maven dependency, use the following:</p>}
+            {hasBomEntry && <p>
+                <>
+                    To avoid version conflicts it is suggested not to depend on a specific version, but use the
+                    {' '}
+                    <a target="_blank" href="https://github.com/jenkinsci/bom#readme" rel="noreferrer">Jenkins plugin BOM</a>
+                    {' '}
+                    and the following depndency snippet:
+                </>
+            </p>}
+            <pre>
```

Comment:
we have the plugin to make things click to copy if its useful here. I was going to say I don't know who would find the maven coords useful till I saw the jira ticket.

https://github.com/jenkins-infra/plugin-site/blob/ef75f3fc7c80916626db32ca7cc78138a3a58103/src/components/InstallViaCLI.jsx#L8